Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN mkdir /stubs
 RUN apk -U --no-cache add git protobuf
 
 RUN go get -u -v github.com/golang/protobuf/protoc-gen-go \
+    github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway \
```

Comment:
First I need to know whether this is a common package or not, otherwise I'll find a way how to add this kind of custom dependecies

Reply:
Another option would be to get the same protos from this repo instead: https://github.com/googleapis/api-common-protos 

These are pretty common protos advocated by Google as part of their API design guide, so it's used by a lot of folks.

A more generic solution would be to have an option to pass in a specific protodir (that's mounted on the image), instead of just defaulting the protodir to the parent directory of the mocked proto file, this lets users be able to pull in all the dependencies in their entire proto project if they want.

Reply:
I just created pr #28 to cover this case. please check it out. with that, user should be able to derive their own dockerfile and install their own dependencies then pass the custom protos with `--imports` arg